lib/entries.js

```diff
@@ -101,14 +101,21 @@ function storage(name, storage, pushover) {
     // TODO: implement
   }
 
-  function remove (fn) {
-    // TODO: implement
+  function remove (query, next) {
```

Comment:
This is not implemented in the API but just for the tests to clean up afterwards.